lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -71,6 +71,13 @@ public void onAuthUpdated() {
 	}
 
 	/**
+	 * Authentication error occurred
+	 */
+	public void onAuthError(ErrorInfo errorInfo) {
```

Comment:
No we can't because these methods are accessed across our subpackages (realtime, rest, transport etc)